### ISSUE TITLE
Uploading on Purchase Confirmation page only works immediately after checkout

### ISSUE THREAD

**Issue by mintplugins** [easydigitaldownloads/edd-upload-file#38]
If you set the Upload location to be on the Receipt page (or Purchase Confirmation page), it works immediately after checking out. 

<img width="676" alt="screen shot 2017-11-30 at 8 56 23 am" src="https://user-images.githubusercontent.com/1638128/33434297-e3961afa-d5ac-11e7-860a-55edb1e4accd.png">

However, if you want to re-visit to upload more files later, and go to your purchase history and click "View Details and Downloads", it loads the purchase confirmation page using the "payment_key" variable in the URL like this:

```
http://edd2.dev/upload-file/checkout/purchase-confirmation/?payment_key=99b12efa6316f4d106b0f72f41feee3f
```

But the upload form is not there in this scenario.

**Comment by pippinsplugins** [comment]
@SDavisMedia want to tackle this?

**Comment by SDavisMedia** [comment]
FUN! Yes, I will tackle this.

**Comment by SDavisMedia** [comment]
@pippinsplugins @mintplugins Here's my stab at it. I looked around and can't see a reason why [this restriction](https://github.com/easydigitaldownloads/edd-upload-file/pull/39/files#diff-72e9d66fff109b9954140a9511f2f9e5L49) was necessary. That's all that blocked the functionality.

For my testing on a separate test site from where I did my dev, I had to merge #27 to see it in action without bugs. But even without that, it appears to be working as expected. So, to be clear, `issue/38` does not include `issue/27` and is ready for testing.
